index.bs

```diff
@@ -229,6 +227,39 @@ The <dfn>AV1MultiFrameSampleGroupEntry</dfn> documents samples that contain mult
 class AV1MultiFrameSampleGroupEntry extends VisualSampleGroupEntry('av1m') {
 }
 ```
+AV1 Forward Access sample group entry {#forwardaccesssamplegroupentry}
+----------------------------------------------------------------
+
+### Definition ### {#forwardaccesssamplegroupentry-definition}
+
+<pre class='elementdef'>
+	Group Type: av1f
+	Container: Sample Group Description Box ('sgpd')
+	Mandatory: No
+	Quantity: Zero or one.
+</pre>
+
+
+### Description ### {#forwardaccesssamplegroupentry-description}
+
+The <dfn>AV1ForwardAccessSampleGroupEntry</dfn> documents samples that contain multiple frames.
+
+
+### Syntax ### {#forwardaccesssamplegroupentry-syntax}
+
+```
+class AV1ForwardAccessSampleGroupEntry extends VisualSampleGroupEntry('av1f') {
+  unsigned int(1) initial_display;
+  unsigned int(7) fwd_distance;
+}
+```
+
+### Semantics ### {#forwardaccesssamplegroupentry-semantics}
+
+The <dfn>initial_display</dfn> fields indicates if the processing of this sample will produce an output frame or not.
```

Comment:
This is for a sample group, not sample. Did you intend to mean whether *any* sample in the group will produce an output frame?

Reply:
+1. Shouldn't it be something like:

indicates if the processing of the first sample in the group will produce an output frame or not

Reply:
A sample group is the mechanism to attach the same property to a set of samples. In this case, each sample marked in the sample group produces an output frame (or not) and is followed N frames after by a show existing frame. Can you suggest better text?

Reply:
@tomfinegan The concept of sample group is not used to describe a set of consecutive samples. There is no notion of first. All samples in a sample group have the same characteristics.

Reply:
OK, this makes more sense knowing that. Thanks.

Reply:
OK. I was a bit confused because I thought it was normally mandatory for all non-fwd-kf sample groups to output a frame, so it didn't seem important to include this bit if it's already labeled av1f. Can't you assume it's 0 if your sample is av1f?

Reply:
Because the KF has show_frame = 0 it has to be bundled in the TU/Sample with another frame that actually is showable. That frame may or may not be decodable with the preceding key frame only. That bit was for that. There are also complex configurations using INTRA ONLY FRAMEs that could be used to achieve the same effect with show_frame = 0 or 1. That's why the sample group is not called ForwardKeyFrame as I think it can be used also in different configurations.

Reply:
OK. I think it now makes sense why the bit is there, but not happy with the name or description. I will try to come up with something better, but it doesn't have to block this pr.